okapi/views/changelog.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace okapi\views\changelog;
+
+use Exception;
+use okapi\Okapi;
+use okapi\Settings;
+use okapi\OkapiRequest;
+use okapi\OkapiHttpResponse;
+use okapi\views\menu\OkapiMenu;
+
+class View
+{
+    public static function call()
+    {
+        require_once($GLOBALS['rootpath'].'okapi/views/menu.inc.php');
+
+        # TODO: load and cache from OKAPI repo
+        # TODO: verify XML scheme
+        $changes = simplexml_load_file('https://raw.githubusercontent.com/following5/okapi/feature/changelog/etc/changes.xml');
+
+        $unavailable_changes = array();
+        $available_changes = array();
+
+        foreach ($changes->change as $change) {
+            if ($change['version'] > Okapi::$version_number)
```

Comment:
I would also add a safeguard here: there's no version or date, skip the entry. (In case one forgets to run `update_changes.php`.)